core-primitives/extrinsics-factory/src/lib.rs

```diff
@@ -123,17 +134,24 @@ where
 					additional_extrinsic_params,
 				);
 				let xt = compose_extrinsic_offline!(&self.signer, call, extrinsic_params).encode();
+				trace!(
+					"encoded extrinsic with nonce {}: 0x{}",
+					nonce_value,
+					hex::encode(xt.clone())
```

Comment:
I'd remove this trace as it introduces a clone

Reply:
the idea here is to format the encoded call exactly the way it could be copy-pasted and inspected with js/apps. So I'd rather leave it as is - unless you have a way to achieve exactly the same without the clone